app/http.php

```diff
@@ -302,6 +303,60 @@ function createDatabase(App $app, string $resourceKey, string $dbName, array $co
 
                 $dbForPlatform->createCollection('bucket_' . $bucket->getInternalId(), $attributes, $indexes);
             }
+
+            if ($dbForPlatform->getDocument('buckets', 'screenshots')->isEmpty() &&
+                !$dbForPlatform->exists($dbForPlatform->getDatabase(), 'bucket_2')) {
```

Comment:
Can we do something better here?

Reply:
I do same as lines above.

In my opinion first check is enough, so if you are good with it, I can remove second part of if from both here, and default bucket above

Reply:
I referring to the use of the ID this way.

Reply:
Me too. To allow us to merge, removing this intenralID check from both existing scenario, and my new one